docs/setup/building.md

```diff
@@ -0,0 +1,106 @@
+# Building and Running Tests
```

Comment:
I would prefer a separate section for Building instructions since this title could mislead people to think that this doc is about building tests and running them.